src/Sylius/Component/Core/Model/OrderInterface.php

```diff
@@ -120,10 +120,6 @@ public function setCustomerWithAuthorization(?BaseCustomerInterface $customer):
 
     public function isCreatedByGuest(): bool;
 
-    public function getCreatedByGuest(): bool;
-
-    public function setCreatedByGuest(bool $createdByGuest): void;
-
```

Comment:
I know this removal is obvious, due to the deprecation of methods in the implementing class, but I still wonder if we should deprecate these interface methods first in Sylius 1.13

Reply:
https://github.com/Sylius/Sylius/pull/15376